src/main/java/duke/ValidCommand.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import duke.exception.AmbiguousInputException;
+
+/**
+ * ValidCommand enum represents the possible command type.
+ */
+public enum ValidCommand {
```

Comment:
I like your use of enums to simplify your program and communicate your ideas more clearly!